src/examples/pBanach.lean

```diff
@@ -0,0 +1,58 @@
+import challenge_notations
+import challenge_prerequisites
+
+/-!
+
+This file explains how the condensed abelian group associated to a p-Banach space `V` behaves.
+
+-/
+
+noncomputable theory
+
+open_locale liquid_tensor_experiment nnreal zero_object
+open liquid_tensor_experiment category_theory category_theory.limits opposite
+
+variables (p : ℝ≥0) (V : pBanach.{0} p) (S : Profinite.{0})
+
+section pBanach
+
+example : topological_add_group V := infer_instance
+
+example : module ℝ V := infer_instance
+
+example : has_continuous_smul ℝ V := infer_instance
+
+example : complete_space V := infer_instance
+
+example : separated_space V := infer_instance
+
+/-- A p-Banach space has a norm. -/
+def pBanach.has_norm : has_norm V :=
+(p_banach.exists_p_norm V.p_banach').some.to_has_norm
```

Comment:
I'm not sure if I like this. The norm is highly non-canonical. If we do this, we should explain that. I agree that it gives nicer notation down-stairs though.

Reply:
I explained that this is a choice in the comments. I think it should now be clear enough that we're really making a choice here.